test/system/252-quadlet.bats

```diff
@@ -467,7 +468,7 @@ EOF
 [Container]
 Image=$IMAGE
 SecurityLabelDisable=true
-Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | socat -u STDIN unix-sendto:\$NOTIFY_SOCKET; top"
+Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | top"
```

Comment:
this also should remove the `echo "READY=1" |` part

Reply:
Sorry, somehow I misread '|' as '||' and kept it